static/elements/chromedash-color-status.mjs

```diff
@@ -0,0 +1,34 @@
+import { LitElement, html } from 'lit-element';
```

Comment:
why .mjs? Browsers don't care about the extension.

Reply:
I personally don't mind the mjs extension since it conveys that the js is in a module but I'm fine either way.

Reply:
I saw this practice from https://philipwalton.com/articles/deploying-es2015-code-in-production-today/. But no strong preference at all. I'll just change back to .js